spec/menu_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'menu'
+describe Menu do
+
+it 'can create a new menu with list of dishes' do
+menu = Menu.new
+expect(menu).to be_a_kind_of(Menu)
+end
+
+it 'can list all the dishes in the menu with prices' do
```

Comment:
Good unit testing here.

Please be careful about indentation - this can make a difference for readability.